contracts/src/v0.8/dev/interfaces/OCR2DRRegistryInterface.sol

```diff
@@ -0,0 +1,156 @@
+   */
+  function getRequestConfig()
+    external
+    view
+    returns (
+      uint16,
+      uint32,
+      bytes32[] memory
+    );
+
+  /**
+   * @notice Initiate the billing process for an OCR2DR request to the DON
+   * @param keyHash - Corresponds to a particular oracle job which uses
+   * that key for generating the OCR2DR proof. Different keyHash's have different gas price
+   * ceilings, so you can select a specific one to bound your maximum per request cost.
+   * @param subId  - The ID of the OCR2DR subscription. Must be funded
+   * with the minimum subscription balance required for the selected keyHash.
+   * @param minimumRequestConfirmations - How many blocks you'd like the
+   * oracle to wait before responding to the request. See SECURITY CONSIDERATIONS
+   * for why you may want to request more. The acceptable range is
+   * [minimumRequestBlockConfirmations, 200].
+   * @param callbackGasLimit - How much gas you'd like to receive in your
+   * fulfillRequest callback. Note that gasleft() inside fulfillRequest
+   * may be slightly less than this amount because of gas used calling the function
+   * (argument decoding etc.), so you may need to request slightly more than you expect
+   * to have inside fulfillRequest. The acceptable range is
+   * [0, maxGasLimit]
+   * @param data Encoded OCR2DR request data, use OCR2DRClient API to encode a request
+   * @return requestId - A unique identifier of the request. Can be used to match
+   * a request to a response in fulfillRequest.
+   */
+  function sendRequest(
+    bytes32 keyHash,
+    uint64 subId,
+    uint16 minimumRequestConfirmations,
+    uint32 callbackGasLimit,
+    bytes calldata data
+  ) external returns (bytes32);
+
+  /*
+   * @notice Conclude billing for an OCR2DR request
+   * @param proof contains the proof and response
+   * @param rc request commitment pre-image, committed to at request time
+   * @return payment amount billed to the subscription
+   * @dev simulated offchain to determine if sufficient balance is present to fulfill the request
+   */
+  function fulfillRequest(
+    bytes32 requestId,
+    bytes32 keyHash,
+    RequestCommitment memory rc,
+    bytes calldata response,
+    bytes calldata err
+  ) external returns (uint96);
+
+  /**
+   * @notice Create a new OCR2DR subscription.
+   * @return subId - A unique subscription id.
+   * @dev You can manage the consumer set dynamically with addConsumer/removeConsumer.
+   * @dev Note to fund the subscription, use transferAndCall. For example
+   * @dev  LINKTOKEN.transferAndCall(
+   * @dev    address(REGISTRY),
+   * @dev    amount,
+   * @dev    abi.encode(subId));
+   */
+  function createSubscription() external returns (uint64 subId);
+
+  /**
+   * @notice Get a OCR2DR subscription.
+   * @param subId - ID of the subscription
+   * @return balance - LINK balance of the subscription in juels.
+   * @return reqCount - number of requests for this subscription, determines fee tier.
```

Comment:
By reqCount are you referring to the # of HTTP requests?

Reply:
If so, can we change this to `queries`?

Reply:
This is literally the number of request sent through the system. Lifted from VRFv2 docs:

> Tiered fees. We want to reward users with cheaper fees the more they use the product. To do that we added FeeConfig which specifies 5 tiers, each with their own flat fee. We chose a flat fee because fundamentally the offchain computation is fixed and essentially free (and nops get reimbursed for their gas costs directly on-chain).

Reply:
We could probably repurpose this into something more useful to us like you are saying